packages/airnode-node/src/types.ts

```diff
@@ -194,6 +195,7 @@ export interface CoordinatorStateWithApiResponses extends CoordinatorState {
 export interface EVMContracts {
   // TODO: Rename to airnodeRrp for consistency
   readonly AirnodeRrp: string;
+  readonly AirnodeRrpDryRun?: string | undefined;
```

Comment:
The `undefined` is redundant to `AirnodeRrpDryRun` being optional and can be removed.